pkg/operator/ceph/nfs/spec.go

```diff
@@ -134,6 +134,7 @@ func (c *CephNFSController) initContainer(n cephv1.CephNFS, binariesEnvVar v1.En
 	return v1.Container{
 		Args: []string{
 			"ceph",
+			"--ceph-version-name", c.cephVersion.Name,
```

Comment:
I don't think we actually care about ceph-version-name in ganesha containers. We won't do anything different either way. Might be simplest to just drop the changes to this file?

Reply:
Bah, nvm. Dumb suggestion...

We _do_ need to know so we can set up ceph.conf properly.

Reply:
Ahh, I think I see. I think we just need to remove this flag from the daemon container, not the init container for ganesha. Testing a patch with that now.